docs/administration.product-feedback-collection.md

```diff
@@ -0,0 +1,71 @@
+---
+id: administration.product-feedback-collection
+title: Product Feedback Collection
+---
+
+> We are happy to share any of the feedback and product usage data we collect with you for your meshStack.
+> Reach out to your Customer Success representative to get the data.
+
+## In-Product Satisfaction Survey
+
+meshStack might show your users a short survey consisting of three questions:
+1. How satisfied are you with {meshStack}*
+2. When I perform an action in {meshStack}, it works as expected.
+3. I was able to complete the tasks I needed to using {meshStack}.
+
+*Please note that if you have your own rebranded name for meshStack, that will be shown instead.
+
+Below you can see an example of what the survey looks like.
+
+![Example of Satisfaction Survey](assets/feedback-survey-example.png)
+
+### When does the survey show up?
+
+The satisfaction survey will show up under the following conditions:
+
+- The user is **not** a user of the Admin Area.
+- The user has logged in more than three times in the last 30 days.
+- The user has not filled in the survey in the last 30 days.
+- The user did not do anything for the last 25 seconds (to prevent interrupting the user in important flows)
+
+Note that the satisfaction survey is not configurable to be turned off. It is always turned on for each meshStack.
+
+## Product Usage Data with Plausible
+
+In order to continuously improve our product we collect product usage data and feedback.
+Product usage data collection can be explicitly enabled by an Partner after prior confirmation of the customer.
```

Comment:
```suggestion
Product usage data collection can be explicitly enabled by a Partner after prior confirmation of the customer.
```
What kind of Partner is this. Can one configure it as an admin in the front end?

Reply:
I copied this over but this actually sounds shit. I replaced it with something else.